portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -638,4 +638,86 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-8239 | Automation ID: LRAC-11536 | Test Summary: Delete a segment in edit mode"
+	@priority = "2"
+	test DeleteSegmentInEditMode {
```

Comment:
Hey @manuferreira, Could you disable this test from running in the upstream routine? The user will not appear as an known in this routine

Reply:
Ohh nice, thanks for pointing that out! I remembered running it in the release branch but forgot to disable it in the upstream